.github/workflows/docker-vulnerabilities.yaml

```diff
@@ -0,0 +1,74 @@
+name: DockerVulnerabilities
+on:
+  push:
+    branches:
+      - main
+  pull_request: 
+  #schedule:
```

Comment:
Want to have the schedule in or not? What's the goal?

Reply:
That's to force at least one run to see it works. In the next commit, I will make it only run when merged into main and on a cron job.